core/src/main/scala/org/apache/spark/network/netty/NettyBlockTransferService.scala

```diff
@@ -22,8 +22,12 @@ import java.nio.ByteBuffer
 import scala.collection.JavaConverters._
 import scala.concurrent.{Future, Promise}
 import scala.reflect.ClassTag
+import scala.tools.nsc.interpreter.JList
```

Comment:
this should be

```scala
import java.util.{List => JList}
```

(it happens to work b/c of this: https://github.com/scala/scala/blob/v2.12.1/src/repl/scala/tools/nsc/interpreter/package.scala#L36)